components/inputs/select.tsx

```diff
@@ -22,16 +23,26 @@ function useSelect() {
 
   return context;
 }
+type Path =
+  | { variant: 'ui'; path: string }
+  | { variant: 'form'; path?: string };
 
 const Select = ({
-  variant = 'ui',
+  variant,
+  path,
```

Comment:
Please make sure these are in alphabetical order.
```suggestion
  path,
  variant,
```